client/src/components/Form/Form.js

```diff
@@ -1,43 +1,74 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect } from 'react';
+import clsx from 'clsx';
 
 // Material-UI Imports
-import { createStyles, makeStyles } from "@material-ui/core/styles";
-import Button from "@material-ui/core/Button";
-import InputLabel from "@material-ui/core/InputLabel";
-import MenuItem from "@material-ui/core/MenuItem";
-import FormControl from "@material-ui/core/FormControl";
-import Select from "@material-ui/core/Select";
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
 
 // Local imports
-import DistanceSlider from "./DistanceSlider";
-import CheckboxGroup from "./CheckboxGroup";
+import DistanceSlider from './DistanceSlider';
+import CheckboxGroup from './CheckboxGroup';
+
+import data from '../../data/demo.json';

```

Comment:
scripts/seedDB.js has our most up to date seed data.

Reply:
Better to do an api call?

Reply:
Yeah remove this, implement API call to get seeded data.